packages/infra/lib/api-stack.ts

```diff
@@ -362,6 +362,21 @@ export class APIStack extends Stack {
       alarmAction: slackNotification?.alarmAction,
     });
 
+    const bulkUrlSigningLambda = this.setupDocumentBulkSignerLambda({
+      lambdaLayers,
+      vpc: this.vpc,
+      medicalDocumentsBucket: medicalDocumentsBucket,
+      fhirServerUrl: props.config.fhirServerUrl,
+      envType: props.config.environmentType,
+      sentryDsn: props.config.lambdasSentryDSN,
+      alarmAction: slackNotification?.alarmAction,
+      searchEndpoint: ccdaSearchDomain.domainEndpoint,
+      searchIndex: ccdaSearchIndexName,
+      searchUserName: ccdaSearchUserName,
+      searchSecretName: ccdaSearchSecret.secretName,
+      loadBalancerDnsName: props.config.loadBalancerDnsName,
```

Comment:
needs load balancer address but lambda is created before API Service so need to use the loadBalancer address in the config